modules/internal/ChapelSyncvar.chpl

```diff
@@ -134,6 +134,17 @@ module ChapelSyncvar {
       this.isOwned = false;
     }
 
+    proc init=(type ThisType, const other : _syncvar) {
+      if other.type != ThisType then compilerError("init= error");
+      this.init(other);
+    }
```

Comment:
Are these actually used? It looks like the implementation was still calling initCopy for sync vars. Is initCopy calling init= now?

Reply:
good catch, these are left over from an earlier effort to use init=, but I backed away from it when it sort of blew up. I'll remove these and the _singlevar one.